app.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "Crates.io Mirror",
+  "description": "A mirror of crates.io",
+  "repository": "https://github.com/rust-lang/crates.io",
+  "env": {
+      "HEROKU": "1",
+      "MIRROR": "1",
+      "GH_CLIENT_ID": {
+          "value": "",
+          "required": false
+      },
+      "GH_CLIENT_SECRET": {
+          "value": "",
+          "required": false
+      },
+      "GIT_REPO_CHECKOUT": "./tmp/index-co",
+      "GIT_REPO_URL": "https://github.com/rust-lang/crates.io-index.git",
```

Comment:
In theory, yes, and eventually it will need to be in practice, but for the moment it doesn't actually matter since client-side cargo is the only thing that uses the index. I'll update the instructions though so that people have it set for when we do start using it.